{"status": "broken", "name": "`type()` on null relationship", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "FunctionsAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486026301, "historyId": "64f0df23bab719e1b5366f8afb3ecece", "start": 1578486026088, "steps": [{"status": "passed", "start": 1578486026088, "stop": 1578486026292, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486026293, "stop": 1578486026296, "name": "And having executed", "attachments": [{"source": "51c8c442-4461-4032-977b-dd4a14020405-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486026296, "stop": 1578486026298, "name": "When executing query", "attachments": [{"source": "16d48a6f-5669-49df-a42c-96fe71a6adc2-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "1f698ac0-2bc9-4562-b1ef-b3f3db3884b6-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486026300, "start": 1578486026299, "statusDetails": {"message": "ResponseError: RedisGraph does not support OPTIONAL MATCH.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486026301, "stop": 1578486026301, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not support OPTIONAL MATCH.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "4b7febbe-a042-49d8-800f-ca0305f0c5f8"}